src/openzaak/components/zaken/api/serializers/zaken.py

```diff
@@ -775,6 +776,12 @@ def create(self, validated_data):
         # are we re-opening the case?
         is_reopening = zaak.einddatum and not is_eindstatus
 
+        is_deelzaak = (
```

Comment:
Maybe this is slightly clearer, because deelzaken that have a different afleidingswijze will still be closed like regular zaken
```suggestion
        afleidingswijze_deelzaak = (
```